test/Polly.Core.Tests/Utils/Pipeline/CompositePipelineComponentTests.cs

```diff
@@ -97,14 +97,14 @@ public async Task Create_Cancelled_EnsureNoExecution()
     [Fact]
     public async Task Create_CancelledLater_EnsureNoExecution()
     {
-        var executed = false;
+        bool executed = false;
```

Comment:
Just curious, what prompted you to make these various changes from `var`?

Reply:
its the `.editorconfig` behavior setup in the repo and my VS runs code cleanup on save (annoying to toggle off, wish I could set it at the repo level).  happy to tweak the .editorconfig to align with the desired behavior.  Options are:

1. Configure [csharp_style_var_for_built_in_types](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008) to be `var`
2. Turn off [IDE0007 and IDE0008](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#suppress-a-warning)
3. Kick the can down the road and have me toggle off code cleanup 😅

Reply:
If our EditorConfig is wrong can you update it please?

Reply:
I have created [this PR ](https://github.com/App-vNext/Polly/pull/1700)to address the editorconfig issue separately, hope this helps!

Reply:
@martincostello rebased after merging var PR and undid the var mods